sample-queries/sample-queries.json

```diff
@@ -1313,6 +1313,22 @@
         "tip": "This query will create a folder called TestBatchingFolder in your OneDrive and return it back to you via a GET.",
         "skipTest": false
     },
+    {
+        "id": "191dc268-4601-4f57-8533-b8afc16f2f8c",
+        "category": "Microsoft Teams",
+        "method": "POST",
+        "humanName": "create team",
+        "requestUrl": "/v1.0/teams",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-post?view=graph-rest-1.0&tabs=http",
+        "headers": [
+            {
+                "name": "Content-Type",
+                "value": "application/json"
+            }
+        ],
+        "postBody": "{\r\n   \"template@odata.bind\": \"https://graph.microsoft.com/v1.0/teamsTemplates('standard')\",\r\n   \"displayName\": \"Architecture Team\",\r\n   \"description\": \"The team for those in architecture design.\"\r\n}",
```

Comment:
Is there any particular rhyme or reason to using 3-space indentation for these? I searched the current samples, and it seems like it varies from 1 to 3, but most of them tend to be 2-space indentations.

I wonder if there is some sort of style guide or something that we should be following; My guess is that it is just nuance and doesn't matter at all, since there are already varied formats in the current samples.

Reply:
It's funny you mention this because @jerryxihe was wondering about the number of spaces for an indentation too yesterday LOL. Like you said, in the current samples its inconsistent., so we could ask the GE team in Kenya about this. 

To answer your first question, I copied the query from the create team docs and it was using 3 space indentations (so no reason for using 3).